Mathlib/MeasureTheory/Integral/DivergenceTheorem.lean

```diff
@@ -255,8 +250,8 @@ theorem integral_divergence_of_hasFDerivWithinAt_off_countable_aux₂ (I : Box (
 variable (a b : Fin (n + 1) → ℝ)
 
 local notation "face " i => Set.Icc (a ∘ Fin.succAbove i) (b ∘ Fin.succAbove i)
--- local notation "frontFace " i:2000 => Fin.insertNth i (b i)
--- local notation "backFace " i:2000 => Fin.insertNth i (a i)
+local notation "frontFace " i:2000 => Fin.insertNth i (b i)
+local notation "backFace " i:2000 => Fin.insertNth i (a i)
```

Comment:
Please don't merge before I try to use `frontFace`/`backFace` in the morning (or you can search for `i.insertNth (a i)` below).